opencti-platform/opencti-graphql/src/domain/user.js

```diff
@@ -669,7 +670,7 @@ export const userEditField = async (context, user, userId, rawInputs) => {
   const userToUpdate = await internalLoadById(context, user, userId);
   // Check in an organization admin edits a user that's not in its administrated organizations
   const myAdministratedOrganizationsIds = user.administrated_organizations.map((orga) => orga.id);
-  if (!isUserHasCapability(user, SETTINGS_SET_ACCESSES) && isUserHasCapability(user, VIRTUAL_ORGANIZATION_ADMIN)) {
+  if (!isAdminWithoutSettingsSetAccessCapability(user)) {
```

Comment:
should be `if (isAdminWithoutSettingsSetAccessCapability(user))`, without the `!`